### ISSUE TITLE
Only bids in last five minutes

### ISSUE THREAD

**Issue by Bob930** [hunterjm/fifa-autobuyer#133]
Is there any way that one can make the autobuyer bid on cards with more than 5 minutes left? That would be very usefull!

**Comment by hunterjm** [comment]
What is the point in tying up your coins so early?

I guess it can be a configurable setting.

**Comment by Bob930** [comment]
I think it should be configurable setting, because for example I have a player that I am buying for 700 and selling for 900. All the players in the last five minutes already have a bid for 700 so I would have to bid 750. But I saw that the players with around 15min left hadn't had a bid on them yet. Till they reached the last five minutes they always had a bid. I also have 350k so I don't mind tying up my coins so early.

**Comment by Nicorad** [comment]
@Bob930 I agree with you. For the past two years, I made my own AB in C# (I'm not a developper, not handling RPM, lot of crashes, this is why I stop it and use this one), Anyway, during the various tests, I pointed out that it was sometime more efficient to bid at 10, sometimes 15 min. It depends on the player...

**Comment by Bob930** [comment]
So are you planning on making it a configurable setting?

**Comment by celioreyes** [comment]
@Bob930 this should be out tonight.

**Comment by celioreyes** [comment]
@hunterjm https://github.com/hunterjm/fifa-autobuyer/pull/145
